README.md

```diff
@@ -42,6 +42,7 @@ Support [Nick Scialli](https://github.com/nas5w) directly on his repo! [Intervie
 
 | Name         | Location     | Roles                | Citizenship/Visa Requirements | Date Posted <br> mm/dd/yyyy |
 | ------------ | ------------ | -------------------- | ----------------------------- | --------------------------- |
+| [Lockheed Martin](https://www.lockheedmartinjobs.com/job/owego/software-engineer-early-career/694/50929022432) | - Owego, New York <br> - Grand Prairie, TX | ✅ [Software Engineer - Early Career](https://lockheedmartinjobs.com/job/owego/software-engineer-early-career/694/50929022432) ✅ | - | 06/28/2023 |
```

Comment:
This position requires a government security clearance, you must be a US Citizen for consideration.